.drone.yml

```diff
@@ -27,21 +22,44 @@ pipeline:
     commands:
       - set -o pipefail
       - make test | tee 'test.log'
+      - source /get-package-details.sh
+      - echo -n "${VERSION}-${DRONE_BUILD_NUMBER}-${DRONE_COMMIT:0:10},${VERSION},latest" > .tags
```

Comment:
I don't think this includes all of the tags we want.

I also wonder if the script should make this file for us.

Reply:
Do we need all the sem-ver tags? Surely most [or all] cases for an app (with no dependants like this) is just the exact version or latest?
I don't think the script should make the file, I think the file should only be made with the tags each case actually requires, otherwise you'd possibly get unwanted tags.

Reply:
I don't think there's any harm in having the extra tags and I think they are nice to have. It seems to be the standard way of doing things.

It's true that we would generally use the exact (commit hash) version ourselves.

As I think there is a 'correct' way to do this. (i.e. Tag all the things.) I think the script may as well create the file.

I suppose my question is, what is the _harm_ in creating a `2.3` tag when creating `2.3.1-2fe2731`, let's say?